plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -177,6 +177,21 @@ public void testCreateOrReplaceTableAsSelect()
                         .add(new FileOperation(DATA, "no partition", "OutputFile.create"))
                         .build());
 
+        assertFileSystemAccesses(
+                "CREATE OR REPLACE TABLE test_create_or_replace_as_select AS SELECT 2 different_col_name",
```

Comment:
I'm guessing you should create a table (make use of `checkpoint_interval` table property) , add a few inserts / fake deletes to it  and check whether the `CORTAS` statement does indeed write a checkpoint file.

We also need an accuracy test in TestDeltaLakeConnectorTest to verify whether the table is still readable after this scenario.

Reply:
Added `TestDeltaLakeFileOperations#testCreateOrReplaceTableWithCheckpoints` and `TestDeltaLakeConnectorTest#testCreateOrReplaceTableWithCheckpointsIsStillQueryable`